libtiny_ui/src/lib.rs

```diff
@@ -78,6 +78,7 @@ pub enum TabStyle {
     Normal,
     NewMsg,
     Highlight,
+    JoinOrPart,
```

Comment:
This should be between `Normal` and `NewMsg`, see the documentation above the type:

```
// NOTE: Keep the variants sorted in increasing significance, to avoid updating
// style with higher significance for a less significant style (e.g. updating
// from `Highlight` to `NewMsg` in `set_tab_style`).
```

Reply:
Oops, I didn't notice it! I will change it tomorrow!